pallene/to_ir.lua

```diff
@@ -495,10 +497,10 @@ function ToIR:exp_to_value(cmds, exp, _recursive)
                 return ir.Value.Function(id)
 
             elseif cname._tag == "checker.Name.Builtin" then
-                error("not implemented")
+                typedecl.tag_error(cname._tag, "not implemented")
```

Comment:
If the error happens, it should be clear what the type tag was because of the preceding if statement. I think we can continue using `error("not implemented")` here.